assignment.js

```diff
@@ -32,6 +32,26 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+//our base number for multiplication table
+const baseNumber = 11
+
+
+// function getRandomInt(max) {
+//   return Math.floor(Math.random() * Math.floor(max));
+// }
+//
+// const baseNumber = getRandomInt(16)
+
+console.log(`Multiplication table for: ${baseNumber}`)
+
+//for loop to itterate through the first 12 numbers in the multiplication table
+for (let i = 0; i<=12; i++){
```

Comment:
@dkwatson522  nice job using a for loop here, code was well structured and easy to follow. 

An example of an alternative approach to this problem using a while loop [can be found here](https://github.com/jsd20200127/hw-01-js-basics-control-flow-loops-solution/blob/master/assignment.js#L38)